.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: node_js
 node_js:
   - 'node'
-branches:
-  only:
-    - master
 deploy:
   on:
+    branch: master
```

Comment:
Do we even need the `branch` option? According to https://docs.travis-ci.com/user/deployment#conditional-releases-with-on it's ignored when tags is set to true

Reply:
Good point, thanks for looking that up.